Tests/scripts/pkg_dev_test_tasks.py

```diff
@@ -63,8 +61,10 @@ def get_python_version(docker_image):
     print("Detected python version: [{}] for docker image: {}".format(py_ver, docker_image))
     py_num = float(py_ver)
     if py_num < 2.7 or (3 < py_num < 3.4):  # pylint can only work on python 3.4 and up
-        raise ValueError("Python version for docker image: {} is not supported: {}. "
-                         "We only support python 2.7.* and python3 >= 3.4.".format(docker_image, py_num))
+        print_warning("Python version for docker image: {} is not supported: {}. "
```

Comment:
Don't change this. We don't support python 3.4. In such cases we need to change the python docker image.